index.js

```diff
@@ -18,5 +19,17 @@ module.exports = (red, green, blue) => {
 		throw new TypeError('Expected three numbers below 256');
 	}
 
-	return ((blue | green << 8 | red << 16) | 1 << 24).toString(16).slice(1);
+	if (typeof alpha === 'number') {
+		if (alpha < 1 && alpha >= 0) {
+			alpha = Math.round(256 * alpha).toString(16);
+		} else if (alpha > 1 && alpha < 100) {
+			alpha = Math.round(256 * alpha / 100).toString(16);
```

Comment:
I always hate this logic. Stick to a single representation of alpha (I'm always in the camp of 0.0 - 1.0). @sindresorhus might have more to say, but this causes problems, say, if I'm `for` looping from 0 to 50 with a step of 0.1. My outputs are going to be crazy for the first few iterations.

In my opinion, this is trying to get _too_ fancy with your implementation.